openwisp_controller/pki/migrations/0008_passphrase_field_removed.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.0.10 on 2019-01-16 15:25
+
```

Comment:
@shashwat1002 this migration file shouldn't be here, you need to install the development version of `django-x509`, I recently updated the instructions for installing for development, take a look at how to keep your dev env in sync: https://github.com/openwisp/openwisp-controller#installing-for-development

PS: this is the cause of the failure of the travis build.